src/aof.c

```diff
@@ -1503,11 +1503,6 @@ int rewriteAppendOnlyFileRio(rio *aof) {
             } else {
                 serverPanic("Unknown object type");
             }
-
-            /* We can try to release memory quickly to avoid COW
-             * in the child process. */
-            if (server.in_fork_child) dontNeedObject(o);
```

Comment:
why did you remove that?
IIRC at some point i said that i'm ok to ignore AOFRW, if we wanna integrate the advises into the rdb serialization code of each type (so we don't need the dismiss function and the iterations and switch-cases it has).
but since we decided to keep it (second iteration after serialization), why not have it in AOFRW too?

is that because of the dump_size guess work? i suppose we use that for AOF too

Reply:
I misunderstood, let me add. Dump_size just is approximate, even the size in AOF may be more accurate